pkg/cryptoEAS/index.go

```
@@ -0,0 +1,37 @@
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"fmt"
+	"io"
+)
+
+func encryptAES(plaintext string, key []byte) ([]byte, []byte) {
+	block, _ := aes.NewCipher(key)
+	nonce := make([]byte, 12)
+	io.ReadFull(rand.Reader, nonce)
```

golangci-lint (1.64.8)

Error return value of `io.ReadFull` is not checked

(errcheck)